mmdet/core/hook/neptunelogger_hook.py

```diff
@@ -153,12 +157,13 @@ def _add_ground_truth(self, runner):
             return
 
         # Select the images to be logged.
-        eval_image_indices = np.arange(len(self.val_dataset))
-        # Set seed so that same validation set is logged each time.
-        rng = np.random.default_rng(42)
-        rng.shuffle(eval_image_indices)
-        self.eval_image_indices = eval_image_indices[:self.
-                                                     num_eval_predictions]
+        if not self.eval_image_indices:
+            eval_image_indices = np.arange(len(self.val_dataset))
+            # Set seed so that same validation set is logged each time.
```

Comment:
I think we can remove this comment as we would be computing indices only once (so this maybe confusing).